### DISCUSSION TITLE
Global Renovate.yml and Renovate.json

### DISCUSSION THREAD

**Discussion by GabrielAPineiro** [renovatebot/renovate#15197] [Request Help]
Hi, We are using renovate in our project as self hosted, and we have a yml file with some shared configurations. We also have a renovate.json file in each project. We wanted to know if the properties declared in the yml file are merge or overridden by the properties in the project json file. I tried to search something in the docs, but couldn't find anything about it

**Comment by rarkins** [comment]
Anything listed in https://docs.renovatebot.com/configuration-options/ may be overridden by the repo's config file
